Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.9
```

Comment:
Things may have changed a bit but

1. When using pants, no need for virtualenv as it bundles your python app, it's requirement and runtime? into a single executable file
2. See how Docker setup in [twoopstracker](https://github.com/CodeForAfrica/api/tree/main/twoops_tracker/docker).